Libraries/CommonFunctions.psm1

```diff
@@ -348,6 +348,10 @@ function Is-VmAlive {
                     Write-LogErr "Linux VM $($vm.RoleName) failed to boot because of a kernel panic."
                     return "False"
                 }
+                if ($TestPlatform -eq "HyperV") {
+                    Write-LogWarn "Checking if HyperV VM changed IP at boot."
```

Comment:
LogInfo should be used, it is not a warning for this check, not even if the IP has changed.